docs/en/resources/tools/alloydbpg/alloydbpgcreatecluster.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "alloydb-pg-create-cluster"
+type: docs
+weight: 1
+description: >
+  The "alloydb-pg-create-cluster" tool creates a new AlloyDB for PostgreSQL cluster in a specified project and location.
+aliases:
+- /resources/tools/alloydb-pg-create-cluster
+---
+
+## About
+
+The `alloydb-pg-create-cluster` tool creates a new AlloyDB for PostgreSQL cluster in a specified project and location.
+The tool takes the following input parameters:
+	* `project` : The GCP project ID where the cluster will be created.
+    * `clusterId`: A unique identifier for the new AlloyDB cluster.
+    * `location` (optional): The GCP location where the cluster will be created. Default: `us-central1`.
+    * `network` (optional): The name of the VPC network to connect the cluster to. Default: `default`.
```

Comment:
We need to mention that this create a private ip cluster and the permissions required to use this tool